womtool/src/main/scala/womtool/WomtoolMain.scala

```diff
@@ -71,21 +77,23 @@ object WomtoolMain extends App {
     SuccessfulTermination(AstTools.getAst(Paths.get(workflowSourcePath)).toPrettyString)
   }
 
-  def v1upgrade(workflowSourcePath: String): Termination = {
-    import cats.implicits._
-    import common.Checked
-    import wdl.draft3.transforms.ast2wdlom.astToFileElement
-    import wdl.draft3.transforms.parsing.fileToAst
-    import wdl.draft3.transforms.wdlom2wdl.WdlWriter.ops._
-    import wdl.draft3.transforms.wdlom2wdl.WdlWriterImpl.fileElementWriter
-    import wdl.model.draft3.elements.FileElement
-
-    val loader = fileToAst andThen astToFileElement
-    val model: Checked[FileElement] = loader.run(Paths.get(workflowSourcePath))
-
-    model match {
-      case Right(wdlModel) => SuccessfulTermination(wdlModel.toWdlV1)
-      case Left(errorList) => UnsuccessfulTermination(errorList.toList.mkString("[", ",", "]"))
+  def upgrade(workflowSourcePath: String): Termination = {
+    val file = Paths.get(workflowSourcePath)
+    val maybeWdl = WdlNamespace.loadUsingPath(file, None, None)
+
+    maybeWdl match {
+      case Success(wdl) =>
+        val maybeWomBundle = wdlDraft2NamespaceWomBundleMaker.toWomBundle(wdl)
+
+        maybeWomBundle match {
+          case Right(womBundle) =>
+            val fileElement: FileElement = womBundle.toWdlom
+            SuccessfulTermination(fileElement.toWdlV1)
```

Comment:
I have a feeling that the `Draft2LanguageFactory` has a `toBundle` method that might be similar